adapters/pixfuture/params_test.go

```diff
@@ -0,0 +1,41 @@
+package pixfuture
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	require.NoError(t, err, "Failed to fetch the json schema.")
+
+	for _, p := range validParams {
+		assert.NoErrorf(t, validator.Validate(openrtb_ext.BidderPixfuture, json.RawMessage(p)), "Schema rejected valid params: %s", p)
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
```

Comment:
This can also be refactored to use the tests pattern and t.Run.

Reply:
The recommended code refactoring has been completed.